lib/animas/animasSimulator.js

```diff
@@ -202,13 +202,14 @@ exports.make = function(config){
     smbg: function(event) {
 
       if(currSMBG != null && (event.subType === 'manual') && (currSMBG.subType === 'linked')) {
+        var linkedBG = currSMBG.value;
         if(event.units === 'mmol/L') {
           // linked value is always in mg/dL, so we have to convert to
           // mmol/L with one significant digit if we want to compare to
           // Animas mmol/L manual BG value
-          currSMBG.value = (currSMBG.value/18.01559).toFixedNumber(1);
+          linkedBG = (currSMBG.value/18.01559).toFixedNumber(1);
```

Comment:
The only other place is in a local function in the CareLink code: https://github.com/tidepool-org/chrome-uploader/blob/master/lib/carelink/common.js#L134. Since we'll be deprecating that eventually I'll put the function in a common place, but not touch the CareLink driver.